src/main/java/de/tum/in/www1/artemis/service/MailService.java

```diff
@@ -190,8 +190,8 @@ else if (notificationSubject instanceof FileUploadExercise) {
                 context.setVariable(EXERCISE_TYPE, "upload");
             }
         }
+        else
```

Comment:
I think the `else if` part should be in one line here.

Reply:
Yes good catch (I think this is a strange artifact from npm run prettier:write
I extracted both in https://github.com/ls1intum/Artemis/pull/4303